SmartDeviceLinkTests/DevAPISpecs/SDLMenuManagerSpec.m

```diff
@@ -269,6 +269,17 @@ @interface SDLMenuManager()
                     expect(canSendRPC).to(equal(YES));
                 });
 
+                // should queue an open menu operation for a copied submenu cell
+                it(@"should queue an open menu operation for a copied submenu cell", ^ {
```

Comment:
Tests that are added for a bug fix like this should fail before the fix is applied. This one passes.